contrib/scripts/checkpr.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -e
+
 if ! which curl >/dev/null || ! which jq >/dev/null; then
     echo "This tool requires 'curl' and 'jq' utilities to work correctly. Please install them."
```

Comment:
Check for `GITHUB_TOKEN` to be set in the environment as well and error out accordingly.

Reply:
Actually, I cleared my token and tried it and it worked. I updated the commit message.